api/controllers/StartController.cs

```diff
@@ -26,18 +28,12 @@ public StartController(AzureMediaServicesClient client, ConfigurationModel confi
 
         public async Task<StatusChangeOutputModel> StartServicesAsync(InputRequestModel input)
         {
-            // 1. Check the current status of all of the services
-            StatusOutputModel preRunServiceStatus = await StatusService.GetStatusAsync(input);
+            SentrySdk.AddBreadcrumb(message: "Beginning start procedure", category: "start", level: BreadcrumbLevel.Info);
 
-            // 2. Start the Streaming Endpoint
-            if (preRunServiceStatus.StreamingEndpoint.Status == ResourceStatusEnum.Stopped)
-            {
-                await Client.StreamingEndpoints.StartAsync(
-                    resourceGroupName: Config.ResourceGroup,
-                    accountName: Config.AccountName,
-                    streamingEndpointName: input.StreamingEndpoint
-                );
-            }
+            StatusOutputModel preRunServiceStatus = await GetPreRunServiceStatus(input);

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5532272)